.circleci/config.yml

```diff
@@ -115,20 +126,25 @@ jobs:
           keys:
             - yarn-packages-{{ checksum "yarn.lock" }}
       - run:
+          name: Install Dependencies
           command: yarn install --frozen-lockfile
       - save_cache:
           key: yarn-packages-{{ checksum "yarn.lock" }}
           paths:
-            - ~/.cache/yarn
-      - run: mkdir -p ./test-results
-      - run: yarn install-repos
-      - run: yarn fetch-drupal-cache
-      - run: yarn build
+            - ~/.cache
```

Comment:
Why not specify `~/.cache/yarn`? [Example](https://circleci.com/docs/2.0/yarn/)

Are there other things in `.cache` that we need?

Reply:
Good question. We also need `~/.cache/Cypress`.

I did a test run manually caching `~/.cache/yarn` & `~/.cache/Cypress` and it turned out to be the same size as caching `~/.cache`.

Reply:
I don't know if it's guaranteed that it will always only be `yarn` and Cypress. I'd rather be explicit here about what we cache.

Reply:
That's a good point. I'll explicitly add them.